projects/1.19/assets/no-chat-reports/nochatreports/lang/zh_cn.json

```
@@ -0,0 +1,130 @@
+	"gui.nochatreports.signing_mode.on_demand": "按需签名",
+	"gui.nochatreports.signing_mode.always.tooltip": "该模式下，在进入服务器后立刻启用消息签名。该模式与原版客户端的行为相同。",
+	"gui.nochatreports.signing_mode.never.tooltip": "该模式下，消息签名永远禁用；将会忽略服务器发送的签名请求。",
+	"gui.nochatreports.signing_mode.never_forced.tooltip": "该模式下，聊天消息无法签名。可能是玩家在使用离线账号，或者是客户端无法从Mojang服务器拉取签名密钥对。",
+	"gui.nochatreports.signing_mode.prompt.tooltip": "该模式下，聊天消息默认不签名，但如果服务器发送请求，将会显示提示界面，在此可以对该游戏会话启用消息签名。",
+	"gui.nochatreports.signing_mode.on_demand.tooltip": "与“请求玩家决策”类似，但会在服务器请求时自动启用消息签名。",
+
+	"configuration.NoChatReports.config": "No Chat Reports",
+	"configuration.NoChatReports.category.client": "客户端",
+	"option.NoChatReports.enableMod": "启用模组",
+	"option.NoChatReports.enableMod.tooltip": "启用模组在客户端的大多数功能。若在会话中更改，需要重进服务器。",
+	"option.NoChatReports.showNCRButton": "显示No Chat Report按钮",
+	"option.NoChatReports.showNCRButton.tooltip": "在多人游戏界面添加一个按钮，在此可以控制模组的大多数客户端功能。",
+	"option.NoChatReports.defaultSigningMode": "默认签名策略",
+	"option.NoChatReports.defaultSigningMode.tooltip": "设置对要求或允许消息签名的服务器所采用的默认签名策略。",
+	"option.NoChatReports.showReloadButton": "显示重连按钮",
+	"option.NoChatReports.showReloadButton.tooltip": "在多人游戏界面添加一个按钮，在此可以控制重连选项。",
+	"option.NoChatReports.verifiedIconEnabled": "启用可信服务器图标",
+	"option.NoChatReports.verifiedIconEnabled.tooltip": "允许该模组在多人游戏界面中，对有No Chat Report或其他反举报措施的服务器添加额外图标。",
+	"option.NoChatReports.verifiedIconOffsetX": "可信图标横向偏移",
+	"option.NoChatReports.verifiedIconOffsetX.tooltip": "指定“服务器安全”图标的横向偏移。",
+	"option.NoChatReports.verifiedIconOffsetY": "可信图标纵向偏移",
+	"option.NoChatReports.verifiedIconOffsetY.tooltip": "指定“服务器安全”图标的纵向偏移。",
+	"gui.NoChatReports.showEncryptionButtonWarning": "§c下述选项不会禁用聊天加密，只会隐藏配置按钮！",
+	"option.NoChatReports.showEncryptionButton": "显示加密按钮",
+	"option.NoChatReports.showEncryptionButton.tooltip": "若配置为false，仅会在聊天界面中隐藏加密按钮。注意：加密选项本身并不会随之关闭。",
+	"option.NoChatReports.showEncryptionIndicators": "显示加密指示符",
+	"option.NoChatReports.showEncryptionIndicators.tooltip": "控制是否显示加密指示符。加密指示符包括消息左侧的紫色条带，以及§5§l（§r§5ȯ§5§l）§r消息右侧的悬浮图标。",
+	"option.NoChatReports.showServerSafety": "显示安全状况",
+	"option.NoChatReports.showServerSafety.tooltip": "允许客户端在聊天界面的右下角显示服务器的安全状况。鼠标指针指向该图标时，将会显示更多信息。",
+	"option.NoChatReports.showServerSafety.moreInfo": "§b点击此处以了解关于服务器安全状况的更多信息",
+	"option.NoChatReports.hideInsecureMessageIndicators": "隐藏未经验证消息指示符",
+	"option.NoChatReports.hideInsecureMessageIndicators.tooltip": "阻止客户端显示未经验证消息指示符。该指示符包括消息左侧的红色条带。",
+	"option.NoChatReports.hideModifiedMessageIndicators": "隐藏被改动消息指示符",
+	"option.NoChatReports.hideModifiedMessageIndicators.tooltip": "阻止客户端显示被改动消息指示符。该指示符包括消息左侧的黄色条带，以及§7§l（§r§7?§7§l）§r消息右侧的悬浮图标。",
+	"option.NoChatReports.hideSystemMessageIndicators": "隐藏系统消息提示符",
+	"option.NoChatReports.hideSystemMessageIndicators.tooltip": "阻止客户端显示系统消息指示符。该指示符包括消息左侧的灰色条带。",
+	"option.NoChatReports.hideWarningToast": "隐藏警告提示框",
+	"option.NoChatReports.hideWarningToast.tooltip": "移除在连接到禁用§oenforce-secure-profile§r的服务器时，在右上角弹出的提示框。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
	"option.NoChatReports.hideWarningToast.tooltip": "连接到禁用§oenforce-secure-profile§r的服务器时，移除右上角弹出的提示框。",
```
原文：**Removes a warning toast that is displayed when entering servers with disabled §oenforce-secure-profile§r.**